src/connectors/aws_account.py

```diff
@@ -0,0 +1,100 @@
+"""AWS Account List
+Lists your AWS Accounts
+"""
+
+from runners.helpers import db
+from runners.helpers.dbconfig import ROLE as SA_ROLE
+
+import boto3
+import datetime
+
+CONNECTION_OPTIONS = [
+    {
+        'name': 'source_role_arn',
+        'title': "source role arn",
```

Comment:
capitalize and make these something more reasonable

Reply:
resolved